### DISCUSSION TITLE
Problem with telegraf and Input mqtt_consumer

### DISCUSSION THREAD

**Discussion by tottyseinsGit** [jblance/mpp-solar#93] [Q&A] [answered]
Hello I hope that I help  here.
I have the following problem my mqtt data look like this

QPIGS /status /inverter_heat_sink_temperature/value 31
QPIGS /status /inverter_heat_sink_temperature/ unit ° C
QPIGS /status / pv_input_current_for_battery/value 7. 0
QPIGS /status /pv_input_current_for_battery/ unit A
QPIGS /status/ pv_input_voltage/ value 89. 9
QPIGS /status/ pv_input_voltage/unit V
everything is formatted as string

I ve already tried out different things but be that I ve made the software update is no longer possible
how can I customize the data of the mqtt that they are written to the database
I'm using Telegraf 1.18.1 and 
InfluxDB shell version: 1.8.4
and grafana 
the whole ran with an older software version without problem but after a total failure with a necessary reinstallation I get it not to run
I am grateful for any suggestion I sit there now ne whole week apology for this bad english but my school english is already more ale 40 years old and only suitable for everyday use

**Comment by jblance** [comment, marked as answer]
What command / config file are you running?
What does an INFO debug give?

**Reply by jblance** [reply]
Also what is the telegraf input config

**Reply by tottyseinsGit** [reply]
Thank you very much for your quick response but at the same time as your message came . . . I got a light
it’s difficult if you don’t read the description completely !
I think it’s good the whole as sytem.service runs this part I had completely overlooked. I have worked the instructions step by step like a year ago and overlooked that now is a lot more comfortable I got it partly after try and error  method;-) the first values I can see in grafana already
